tests/test_tpa_pipeline.py

```diff
@@ -6,29 +6,17 @@
 
 import unittest
 
-from django.http import HttpResponseRedirect
 import mock
 from pytest import mark, raises
 
-from enterprise.models import (
-    EnterpriseCustomer,
-    EnterpriseCustomerUser,
-    UserDataSharingConsentAudit,
-)
-
-from enterprise.tpa_pipeline import (
-    get_enterprise_customer_for_request,
-    get_ec_for_running_pipeline,
-    get_enterprise_customer_for_sso,
-    active_provider_enforces_data_sharing,
-    active_provider_requests_data_sharing,
-    get_consent_status_for_pipeline,
-    verify_data_sharing_consent,
-    set_data_sharing_consent_record,
-)
+from django.http import HttpResponseRedirect
 
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser, UserDataSharingConsentAudit
+from enterprise.tpa_pipeline import (active_provider_enforces_data_sharing, active_provider_requests_data_sharing,
+                                     get_consent_status_for_pipeline, get_ec_for_running_pipeline,
+                                     get_enterprise_customer_for_request, get_enterprise_customer_for_sso,
+                                     set_data_sharing_consent_record, verify_data_sharing_consent)
```

Comment:
Nit: personally I think the earlier syntax (one imported symbol per line, comma after each one) is easier to read and produces nicer diffs. I see we are using this "cram everything over to the right and put up with ugly diffs" approach in various places in this repo though :)